presto-product-tests/pom.xml

```diff
@@ -198,6 +198,23 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
Makes sense, I've reverted the SpecificSchema's. I have a strong preference towards Specific over Dynamic since the specific errors show up at compile-time, but I can understand that for Presto the generic schema is more common.